lib/constants.js

```diff
@@ -1,34 +1,34 @@
 const { hex } = require('./helpers');
 
 const StakingUintParamType = {
-  MIN_STAKE: hex('MIN_STAK'),
-  MAX_EXPOSURE: hex('MAX_EXPO'),
-  MIN_UNSTAKE: hex('MIN_UNST'),
-  UNSTAKE_LOCK_TIME: hex('UNST_LKT'),
+  MIN_STAKE: hex('MIN_STAK'.padEnd(8, '\0')),
```

Comment:
let's package this as a function  since it occurs so many times

`function codeToHex(code) => hex(code.padEnd(8, '\0');`

or another name you like.